src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -421,6 +421,22 @@ public async Task<IActionResult> SyncTriggers()
                 : StatusCode(StatusCodes.Status500InternalServerError, new { status = result.Error });
         }
 
+        [HttpGet]
+        [Route("admin/host/triggers")]
+        [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
+        [ResourceContainsSecrets]
```

Comment:
Is the `ResourceContainsSecrets` needed for this API? Can you remind me why?

Reply:
This API returns sync triggers payload which contains secrets as well. So treating it the same ways we treat Keycontroller APIs using ResourceContainsSecrets. I have added you to the Teams thread that has more context.